src/Apps/Collect/Components/Filters/ColorFilter.tsx

```diff
@@ -0,0 +1,229 @@
+import { CheckIcon } from "@artsy/palette"
+import { FilterState } from "Apps/Collect/FilterState"
+import { ContextConsumer } from "Artsy/SystemContext"
+import React from "react"
+
+interface Props {
+  filters: FilterState
+}
+
+const CheckmarkPositions = {
```

Comment:
this is nitpicky -- but if this is just an object

`const checkmarkPositions =` 

CapsCase is idomatic for classes and components, but not for anything else.